131 cadenza baseline 1 (#177)
claritychallenge/clarity: recipes/cad1/README.md

# The 1st Cadenza Challenge
Cadenza challenge code for the 1st challenge.

> For more information about the Cadenza Challenge please visit https://cadenzachallenge.com

Reviewer: URL doesn't resolve, as its `.org`, we should also use Markdown links as GitHub renders Markdown files automatically.

```suggestion
For more information about the Cadenza Challenge please visit the [homepage](https://cadenzachallenge.org)
```

Author: I changed this README to contain only the baselines results.

> Cadenza tutorials are now available in https://cadenzachallenge.com.

Reviewer: Is there a specific page for the tutorials or is it the [learning resources](http://cadenzachallenge.org/docs/learning_resources/learning_intro)? If its the learning resources perhaps the following...

```suggestion
[Learning Resources](http://cadenzachallenge.org/docs/learning_resources/learning_intro) are available on the challenge homepage.```

Reviewer: I missed that the URL here is incorrect too, it should be `.org` and not `.com` even if links to the specific tutorials aren't available for inclusion in this PR.

> The tutorials introduce the Cadenza installation, how to interact with Cadenza metadata,

Reviewer: It may be in the wings waiting to be deployed but the learning resources (see above comment) don't include this information. I think it might be useful to link to the specific pages for each of these, perhaps as a bullet list as suggested above.

Reviewer: I can't find where these tutorials are nor where the code for the site (https://cadenzachallenge.org/) resides (none of the repos within this organisation jumped out as being the one).

It might be worth capturing the completion of these pages as an Issue either within Clarity or on the repository for the site if its its under version control/GitHub.

and also provide examples of baseline systems and evaluation tools.

## Data structure

To download data, please visit [here](https://mab.to/zU7TS8jJelkoD).
The data is split into three packages:

> `clarity_CEC2_core.v1_0.tgz` [28 GB],

Reviewer: These could be a bullet list, and could even link to the files directly...

```suggestion
* [`clarity_CEC2_core.v1_0.tgz`](<path_to_file>) [28 GB],
```

Reviewer: Hosting service doesn't provide direct links hence whey these can't be included.
